hugegraph-server/hugegraph-core/src/main/java/org/apache/hugegraph/backend/query/Condition.java

```diff
@@ -345,31 +345,31 @@ public static Relation neq(HugeKeys key, Object value) {
         return new SyspropRelation(key, RelationType.NEQ, value);
     }
 
-    public static Condition in(HugeKeys key, List<?> value) {
+    public static Relation in(HugeKeys key, List<?> value) {
```

Comment:
The return type changes from Condition to Relation break API compatibility. Ensure all calling code has been updated to handle Relation instead of Condition.